torchvision/io/image.py

```diff
@@ -1,15 +1,18 @@
+import ctypes
+import os
+import sys
 from enum import Enum
+from warnings import warn
 
 import torch
 
-from .._internally_replaced_utils import _get_extension_path
+from .._internally_replaced_utils import _load_library
 
 
 try:
-    lib_path = _get_extension_path("image")
-    torch.ops.load_library(lib_path)
-except (ImportError, OSError):
-    pass
+    _load_library("image")
+except (ImportError, OSError) as e:
+    warn(f"Failed to load {globals().get('lib_path','image.pyd')}: {e}")
```

Comment:
nit: does `pyd` file exists for all platforms?

Reply:
Thank you for pointing this out, this is an artifact from a refactor, I guess simple warning it sufficient there